archinstall/lib/nvidia_utils.py

```diff
@@ -0,0 +1,15 @@
+import archinstall
```

Comment:
From within the `archinstall` library use relative imports instead.

Reply:
Hi, yes. I have fixed the code. It now uses relative imports and only imports `storage` from the `storage` module.